docs/en/operation/Performance-Tuning.md

```diff
@@ -214,6 +214,19 @@ Increasing the number of threads can decrease the staleness of the UFS path cach
 but may impact performance by increasing work on the Alluxio master, as well as consuming UFS bandwidth.
 If this is set to 0, the cache is disabled and the `ONCE` setting will behave like the `ALWAYS` setting.
 
+### Metadata Sync
+
+If the content on the UFS are modified without going through Alluxio, Alluxio needs to sync its metadata with the UFS to reflect those changes in Alluxio namespace. 
+The cost of metadata sync scales linearly with the number of files in the directory that is being synced. 
+If metadata sync operation happens frequently on large directories, more threads may be allocated to speed up this process. 
+Two configurations are relevant here. 
+
+'alluxio.master.metadata.sync.concurrency.level' controls the concurrency that is used in a single sync operation.
+Adjust this to 1x to 2x core count on the master node to speed up the speed of metadata sync.
+
+'alluxio.master.metadata.sync.executor.pool.size' controls the number of threads performing sync operations.
```

Comment:
Do you wanna mention the prefetch thread pool?

Reply:
the prefetch thread pool default is usually large enough to handle all situations, that's why I left it out to reduce confusion.